modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -26,7 +26,7 @@ MariaDB 10+
 | Apache 2.4 with xref:installation/manual_installation.adoc#multi-processing-module-mpm[`prefork and mod_php`]
 
 | PHP Runtime
-| 7.2
+| 7.3
```

Comment:
`site.yml` has `recommended-php-version` - can that be substituted here?
And then ask someone if 7.3 is now the recommended version, I think it would be good to recommend PHP 7.3 for anyone who needs to upgrade anyway.

Reply:
Thanks for those two points.